.github/workflows/ci.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
       - name: Codespell test
         uses: codespell-project/actions-codespell@master
         with:
-          skip: vendor, ./lokoctl, *.png, assets/charts/components/*, docs/images/*,
+          skip: ./vendor,./lokoctl,*.png,./assets/charts/components/*,./docs/images/*,./assets/charts/components/cert-manager,./assets/charts/components/rook,./assets/charts/components/prometheus-operator,./assets/charts/components/velero,./assets/charts/components/cluster-autoscaler,./assets/charts/components/contour/crds,./assets/charts/components/openebs-operator/README.md,./assets/charts/control-plane/calico/crds,./docs/images/lokomotive-example.gif
```

Comment:
Doesn't it skip the entire tree? Like we have `assets/charts/components/*` and then specific paths again?

Reply:
I was also confused, why `assets/charts/components/*` didn't work

Reply:
Ah true, I think `./assets/charts/components/*` should be removed, so we are aware of new typos and we have a chance to fix them upstream.

Reply:
But why check them in some and ignore them in others? How is this decided?

Reply:
We ignore projects which are already in the codebase and contain typos. If you let's say update chart X and update brings typos, the correct thing to do is to send patch upstream fixing them and adding it to the list here until it has been fixed upstream, so it does not block merging on our side.

Reply:
What guarantee do we have that upstream won't repeat the mistake again?

Reply:
>What guarantee do we have that upstream won't repeat the mistake again?

- https://github.com/rook/rook/pull/6564
- https://github.com/prometheus-community/helm-charts/pull/322
- https://github.com/projectcontour/contour/pull/3110
- https://github.com/prometheus-operator/prometheus-operator/issues/3654

Also, even if they do, we repeat the process, it's not a big deal.

Reply:
I think we are just creating more work by doing that. We should simply ignore the entire assets directory. We take care of things that are in our control.

The problems from upstream should not add to our mental burden.

Reply:
I disagree. Given that we actively consume those assets, I believe we should at least contribute typo fixes back.

Reply:
Ping @surajssd

Reply:
I don't agree on on the specific skipping we should rather skip the whole assets dir like I mentioned before, because I can see that we are creating more work for ourselves. I don't mean we should not fix the upstream for spellings but should not be at the expense of Lokomotive development resources.

Since I disagree with the change, please go ahead with merging if it is the right thing to do.

Reply:
@iaguis what do you think?